### ISSUE TITLE
No nightly wheel with CUDA 8.0

### ISSUE THREAD

**Issue by yaroslavvb** [tensorflow/tensorflow#3052]
Current nightly Linux wheel seems to be built with CUDA 7.5

Going from 7.5 to CUDA 8.0 gives me about 50% speed in TensorFlow up on 2048x2048 matmul on GTX 980 (2 T ops/sec -> 3 T ops/sec)
```

export url=http://ci.tensorflow.org/view/Nightly/job/nigntly-matrix-linux-gpu/TF_BUILD_CONTAINER_TYPE=GPU,TF_BUILD_IS_OPT=OPT,TF_BUILD_IS_PIP=PIP,TF_BUILD_PYTHON_VERSION=PYTHON2,label=gpu-linux/143/artifact/pip_test/whl/tensorflow-0.8.0-cp27-none-linux_x86_64.whl
pip install --upgrade $url

# check that it has GPU support
# note, I see performance gain of 50% on matmul going from 7.5 to 8.0
ldd ~/tensorflow/local/lib/python2.7/site-packages/tensorflow/python/_pywrap_tensorflow.so | grep libcudart
 	libcudart.so.7.5 => /usr/local/cuda-7.5/lib64/libcudart.so.7.5 (0x00007f4290bec000)

```

**Comment by aselle** [comment]
Checking on status of pip 8.0...

**Comment by vrv** [comment]
We'll switch our binaries to cuda 8.0 once it's out of RC -- users can always build from sources to get 8.0 goodness.

**Comment by tobegit3hub** [comment]
Hi @yaroslavvb . The [link](http://ci.tensorflow.org/view/Nightly/job/nigntly-matrix-linux-gpu/TF_BUILD_CONTAINER_TYPE=GPU,TF_BUILD_IS_OPT=OPT,TF_BUILD_IS_PIP=PIP,TF_BUILD_PYTHON_VERSION=PYTHON2,label=gpu-linux/143/artifact/pip_test/whl/tensorflow-0.8.0-cp27-none-linux_x86_64.whl) of your package is 404 now.

Can you provide another package for cuda 8.0? We would like to test with Tesla M40 and cuda 8.0.

**Comment by yaroslavvb** [comment]
That was link for cuda 7.5 version, you have to build from source if you
want 8

**Comment by yaroslavvb** [comment]
BTW, CUDA 8.0 has been officially released

**Comment by vrv** [comment]
Cool!  https://github.com/nvidia/nvidia-docker hasn't yet been updated with the next version, which is what we use in our CI to build the releases.

@3XX0, @benbarsdell -- do you have an ETA when this will be done?

(We just minted r0.11,  so this will probably be part of r0.12).

**Comment by 3XX0** [comment]
Yes sorry for that, we need approval and QA the new images before publishing them, this might take a while.

**Comment by shubhamjain0594** [comment]
Installing with source is quite a task. So please let us know asap when it is done. Thank You.

**Comment by 3XX0** [comment]
CUDA 8.0 images are now available!

**Comment by shubhamjain0594** [comment]
Where can I find them?

**Comment by steven200796** [comment]
@3XX0 where can these images be found

**Comment by Dawars** [comment]
It's probably cuda images, not tensorflow

**Comment by aselle** [comment]
Yes, @Dawars is right, @3XX0 is referring to  CUDA nvidia-docker images being available for 8.0. We are looking to release CUDA 8.0 binaries,for rc1. Until then you can build from source.

**Comment by loveJasmine** [comment]
any can help?
I need tf0.8 with cuda8
